build.gradle

```diff
@@ -61,8 +61,8 @@ project.ext.externalDependency = [
     'dgraph4j' : 'io.dgraph:dgraph4j:21.03.1',
     'dropwizardMetricsCore': 'io.dropwizard.metrics:metrics-core:4.2.3',
     'dropwizardMetricsJmx': 'io.dropwizard.metrics:metrics-jmx:4.2.3',
-    'ebean': 'io.ebean:ebean:11.33.3',
-    'ebeanAgent': 'io.ebean:ebean-agent:11.27.1',
+    'ebean': 'io.ebean:ebean:12.16.1',
```

Comment:
Just curious - why are these changes in here?

Reply:
Ebean upgrade is due to a bug in the version that we are currently on that prevents troubleshooting any connection issues. They have a pretty glaring bug that results in every single error that happens at start-up to result in a NPE that masks the real error. Per the Ebean website, there are no necessary migration steps between 11 & 12 for us: https://ebean.io/docs/upgrading/

Reply:
perfect - thank you. we've encountered this bug many times.

Reply:
Looks like it caused a weird issue on this PR's tests even though on acryl-main it didn't so we rolled it back unfortunately :(